kagami/webui/src/Adpage/announcement/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import styles from './styles.module.scss';
+import MarkdownIt from 'markdown-it';
+// 安装 @types/markdown-it 后，可以导入类型
+
+
+interface Announcement {
+  id: number;
+  data: {
+    title: string;
+    content: string;
+    date: string;
+  };
+}
+
+const md = new MarkdownIt();
+
+const AdSection: React.FC = () => {
+  const [announcement, setAnnouncement] = useState<Announcement[]>([]); // 状态变量与更新函数，状态变化会触发重新渲染
+
+  useEffect(() => {
+    // 从后端获取公告数据
+    fetch('http://127.0.0.1:4523/m1/5454758-5129914-default/api/announcement')
```

Comment:
Please don't hard-code test backend. Can you use configurable endpoint settings?